src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -368,6 +368,8 @@ export class VisualizeEmbeddable
   };
 
   private async updateHandler() {
+    getSearch().session.start();
```

Comment:
@lizozom, I am not sure this is correct. As I understand this start a new session for each viz. embeddable on a dashboard. I think this is not expected?